web/ui/src/Services/AppsController.js

```diff
@@ -6,78 +6,41 @@
+        "$modalService", "$translate", "$timeout",
+        "$cookies", "servicesFactory", "miscUtils",
     function($scope, $routeParams, $location, $notification, resourcesFactory, authService, $modalService, $translate, $timeout, $cookies, servicesFactory, utils){
         // Ensure logged in
         authService.checkLogin($scope);
 
-        //constantly poll for apps that are in the process of being deployed so we can alert the user
-        $scope.deployingServices = [];
-        var lastPollResults = 0;
-        var pollDeploying = function(){
-            resourcesFactory.get_active_templates(function(data) {
-                if(data === "null"){
-                    $scope.services.deploying = [];
-                }else{
-                    $scope.services.deploying = data;
-                }
+        // alias hostname instead of using localhost or IP
+        $scope.defaultHostAlias = $location.host();
 
-                //if we have fewer results than last poll, we need to refresh our table
-                if(lastPollResults > $scope.services.deploying.length){
-                    servicesFactory.update();
-                }
-                lastPollResults = $scope.services.deploying.length;
-            });
-        };
-        $scope.name = "apps";
-        $scope.params = $routeParams;
-        $scope.resourcesFactory = resourcesFactory;
-
-        $scope.defaultHostAlias = location.hostname;
-        var re = /\b(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\b/;
-        if (re.test(location.hostname) || location.hostname === "localhost") {
-            $.getJSON("/hosts/defaultHostAlias", "", function(data) {
-                $scope.defaultHostAlias = data.hostalias;
-            });
-        }
-
-        $scope.breadcrumbs = [
-            { label: 'breadcrumb_deployed', itemClass: 'active' }
-        ];
-
-        $scope.services = utils.buildTable('PoolID', [
-            { id: 'Name', name: 'deployed_tbl_name'},
-            { id: 'Description', name: 'deployed_tbl_description'},
-            { id: 'Health', name: 'health_check', hideSort: true},
-            { id: 'DeploymentID', name: 'deployed_tbl_deployment_id'},
-            { id: 'PoolID', name: 'deployed_tbl_pool'},
-            { id: 'VirtualHost', name: 'vhost_names', hideSort: true}
-        ]);
-
-        $scope.templates = utils.buildTable('Name', [
-            { id: 'Name', name: 'template_name'},
-            { id: 'ID', name: 'template_id'},
-            { id: 'Description', name: 'template_description'}
-        ]);
-
-        $scope.click_app = function(id) {
+        // redirect to specific service details
+        $scope.routeToService = function(id) {
             $location.path('/services/' + id);
         };
 
-        $scope.click_pool = function(id) {
+        // redirect to specific pool
+        $scope.routeToPool = function(id) {
```

Comment:
There are several places where the code builds URLs, and it's all kind of scattered around - some in resourcesFactory, some in the resource-specific file like PoolsController and some in other places like here and HostController.

Another good cleanup item (maybe another PR?) would to consolidate all of methods like `routeToPool` in one place resourcesFactory perhaps?   It's unlikely to happen for 1.1, but we've talked about reorganizing the backend URL namespace or standing up a separate node server. When that happens, it will be easier to update the UI code if the URL manipulation code is in fewer places